app/controllers/Checkout.scala

```diff
@@ -174,7 +176,7 @@ object Checkout extends Controller with LazyLogging with CatalogProvider {
     implicit val resolution: TouchpointBackend.Resolution = TouchpointBackend.forRequest(NameEnteredInForm, tempData)
     implicit val tpBackend = resolution.backend
     val idUserOpt = authenticatedUserFor(request)
-    val srEither = tpBackend.subsForm.bindFromRequest
+    val srEither: \/[Seq[FormError], SubscribeRequest] = tpBackend.subsForm.bindFromRequest
```

Comment:
I know this hasn't been changed here, but 'srEither' and 'sr' are not great names...